packages/sdk/src/react/StoryBoost/StoryBoost.tsx

```diff
@@ -52,10 +52,10 @@ const StoryBoostFooter = styled.div`
 
 export interface StoryBoostProps {
   boostId: string;
-  isLoggedIn: boolean;
+  civilUser?: any;
   userAddress?: string;
-  userEmail?: string;
   handleLogin(): void;
+  handleLogout(): void;
```

Comment:
None of these props are being used, I guess I left them in or merge added them and/or some new ones back.

Reply:
Opps I missed these when I merged